src/librustc_metadata/cstore_impl.rs

```diff
@@ -203,11 +203,6 @@ provide! { <'tcx> tcx, def_id, other, cdata,
             DefId { krate: def_id.krate, index }
         })
     }
-    proc_macro_decls_static => {
-        cdata.root.proc_macro_decls_static.map(|index| {
```

Comment:
I don't understand, this was supposed to be used to compute the symbol name, what happened to that?